ui/app/AppLayouts/Communities/panels/TokenHoldersList.qml

```diff
@@ -0,0 +1,261 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+import QtQuick.Controls 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+
+import SortFilterProxyModel 0.2
+
+import utils 1.0
+import shared.controls 1.0
+
+StatusListView {
+    id: root
+
+    readonly property alias sortBy: d.sortBy
+    readonly property alias sorting: d.sorting
+
+    property bool isSelectorMode: false
+
+    signal selfDestructAmountChanged(string walletAddress, int amount)
+    signal selfDestructRemoved(string walletAddress)
+
+    currentIndex: -1
+
+    component ColumnHeader: StatusSortableColumnHeader {
+        id: columnHeader
+
+        leftPadding: 0
+        rightPadding: 0
+
+        Connections {
+            target: d
+
+            function onResetOtherHeaders(header) {
+                if (header !== columnHeader)
+                    columnHeader.reset()
+            }
+        }
+
+        onClicked: {
+            d.resetOtherHeaders(this)
+
+            if (sorting === StatusSortableColumnHeader.Sorting.Ascending)
+                d.sorting = Qt.AscendingOrder
+            else if (sorting === StatusSortableColumnHeader.Sorting.Descending)
+                d.sorting = Qt.DescendingOrder
+        }
+    }
+
+    component NumberCell: StatusBaseText {
+        horizontalAlignment: Qt.AlignRight
+
+        font.weight: Font.Medium
+        font.pixelSize: 13
+
+        color: Theme.palette.baseColor1
+        elide: Qt.ElideRight
+    }
+
+    QtObject {
+        id: d
+        property int sortBy: TokenHoldersProxyModel.SortBy.None
+        property int sorting: Qt.DescendingOrder
+        property var selectedTokenAmount: new Map();
+        property var selectedTokenChecked: new Map();
+
+        readonly property int red2Color: 4
```

Comment:
You can remove this magic number; it's the `red2` named color from `StatusColors.colors`

Reply:
done